src/stores/gameStore.ts

```diff
@@ -15,26 +15,53 @@ export class GameStore {
 
   gameRound = 1;
 
-  bet = 0;
+  // bet = 0;
+
+  playerBet = 0;
+
+  tieBet = 0;
+
+  bankerBet = 0;
 
   winner = "";
 
   constructor() {
     makeAutoObservable(this, {}, { autoBind: true });
   }
 
+  get totalBet() {
+    return this.playerBet + this.tieBet + this.bankerBet;
+  }
+
   setGameStage(gameStage: GameStage) {
     this.gameStage = gameStage;
   }
 
-  setCurrentBet(bet: number) {
-    this.bet += bet;
+  addToPlayerBet(bet: number) {
+    this.playerBet += bet;
```

Comment:
Player's money aren't taken into account? 🙂 

What I noticed is that you can just start game without setting your name and initial money amount. Or you'd like to fix it in upcoming PRs?

Reply:
upcoming PR